docker-rancher/azuredeploy.json

```diff
@@ -0,0 +1,163 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Username for the Virtual Machine."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+	"deployPrefix": {
```

Comment:
Let's prefix this with unique since this is being used for domain label and that tends to cause a lot of deployment failures due to folks entering stuff like "test", "testDeployment" etc.